javascript-style-guide.md

```diff
@@ -5,23 +5,26 @@ The following document outlines a style guide for JavaScript programs.
 # Prelude
 
 StoryCloud's JavaScript Style Guide extends Douglas Crockford's
-[Code Conventions for the JavaScript Programming Language](http://javascript.crockford.com/code.html). Everything
-in that style guide also applies to this style guide. Please familiarize
-yourself with it.
+[Code Conventions for the JavaScript Programming Language][]. Everything in that
```

Comment:
Rather than memorizing "everything there except what's here", I'd prefer to see everything all in one place. Plus, his site hasn't been updated in awhile and has some misleading examples (his section on function declarations, for instance).